plausible/umbrel-app.yml

```
@@ -0,0 +1,27 @@
+manifestVersion: 1
+id: plausible
+icon: https://svgshare.com/i/y30.svg
+category: developer
+name: Plausible Analytics
+version: "2.1.1"
+tagline: Easy to use and privacy-friendly Google Analytics alternative
+description: >-
+  Plausible is intuitive, lightweight and open source web analytics.
+  No cookies and fully compliant with GDPR, CCPA and PECR.
+  Made and hosted in the EU, powered by European-owned cloud infrastructure.
+releaseNotes: ""
+developer:  Plausible Community Edition
+website: https://plausible.io/
+repo: https://github.com/plausible/community-edition/
+support: https://github.com/plausible/community-edition/issues
+port: 9091
+gallery:
+  - https://plausible.io/assets/images/privacy-focused-web-analytics.png
+  - https://i.imgur.com/fUtZ3qJ.png
+  - https://i.imgur.com/fUtZ3qJ.png
```

We'll host the gallery assets here: https://github.com/getumbrel/umbrel-apps-gallery

So this can be changed to:
```suggestion
gallery:
  - 1.jpg
  - 2.jpg
  - 3.jpg
```